xtevent/xtevent.sthlp

```diff
@@ -293,20 +314,17 @@ the cohort variable. {opt save} adds the new control cohort variable to the data
  is not specified.

```

Comment:
I think we could change it for:
"{opt control_cohort(create)} is the default if **{opt cohort(create)}** is specified but {opt control_cohort} is not specified."

Reply:
Thanks, agreed, please implement @Constantino-Carreto-Romero

Reply:
changed in https://github.com/JMSLab/xtevent/commit/ebb8939bedbf7026f74e8de0870fbf19685b22c6